src/main/java/uk/gov/companieshouse/pscfiling/api/config/InterceptorConfig.java

```diff
@@ -22,9 +22,8 @@
 @PropertySource("classpath:validation.properties")
 public class InterceptorConfig implements WebMvcConfigurer {
 
-    static final String[] TRANSACTIONS_LIST = {
-            "/transactions/{transaction_id}/persons-with-significant-control/{pscType:"
-                    + "(?:individual|corporate-entity|legal-person)}"
+    static final String[] INTERCEPTOR_PATHS_LIST = {
+            "/transactions/{transactionId}/persons-with-significant-control/**"
```

Comment:
See https://github.com/companieshouse/psc-filing-api/pull/45
We need to retain this limitation in the regex to avoid a NPE in the Transaction Interceptor.
Using an incorrect endpoint will still get caught and return a 404 as it ought.

Reply:
re-instated regex